src/main/docs/guide/httpServer/serverConfiguration/threadPools/blockingOperations.adoc

```diff
@@ -10,4 +10,22 @@ micronaut:
       nThreads: 75
 ----
 
-The above configuration creates a fixed thread pool with 75 threads.
\ No newline at end of file
+The above configuration creates a fixed thread pool with 75 threads.
+
+== Virtual Threads
+
+Since Java 19, the JVM includes experimental support for virtual threads ("project loom"). As it is a preview feature, you need to pass `--enable-preview` as a JVM parameter to enable it.
+
+The Micronaut framework will detect virtual thread support and use it for the executor named `blocking` if available. If virtual threads are not supported, this executor will be aliased to the `io` thread pool.
+
+To use the `blocking` executor, simply mark e.g. a controller with `ExecuteOn`:
+
+.Configuring the Server I/O Thread Pool
+[source,java]
+----
+@ExecuteOn(TaskExecutors.BLOCKING)
```

Comment:
Should we recommend users to use?

`@ExecuteOn(TaskExecutors.BLOCKING)`

instead of: 

`@ExecuteOn(TaskExecutors.IO)`

Even if they are not using Java 19 yet.

Reply:
i'm honestly not sure which is better. benchmarks show the IO executor still being faster.